leukeleu_django_gdpr/management/commands/gdpr.py

```diff
@@ -1,162 +1,43 @@
-        "description": description,
-        "help_text": str(help_text),
-        "required": required,
-        "pii": None,
-    }
-
-
-class Serializer:
-    def __init__(self, exclude_list=None, include_list=None):
-        self.models = {}
-        self.exclude_list = exclude_list or []
-        self.include_list = include_list or []
-
-    def generate_models_list(self):
-        self.models = dict(
-            chain.from_iterable(
-                self.handle_app(app_config)
-                for app_config in apps.get_app_configs()
-                if self.should_include(app_config)
-            )
-        )
-
-    def should_include(self, o):
-        s = ""
-        if isinstance(o, AppConfig):
-            s = o.label
-        elif isclass(o) and issubclass(o, Model):
-            s = o._meta.label
-        elif isinstance(o, (Field, GenericForeignKey)):
-            s = f"{o.model._meta.label}.{o.name}"
-
-        include = (
-            not object_matches_exclude_types(o)
-            and not (getattr(o, "auto_created", False))
-            and not any(re.fullmatch(pattern, s) for pattern in self.exclude_list)
-        ) or any(re.fullmatch(pattern, s) for pattern in self.include_list)
 
-        return include
+from leukeleu_django_gdpr.gdpr import Serializer, pii_stats, read_data
 
-    def handle_model(self, model):
-        return (
-            model._meta.label,
-            {
-                "name": str(model._meta.verbose_name).title(),
-                "fields": {
-                    field.name: serialize_field(field)
-                    for field in model._meta.get_fields()
-                    if self.should_include(field)
-                },
-            },
-        )
 
-    def handle_app(self, app_config):
-        for model in app_config.get_models():
-            if self.should_include(model):
-                yield self.handle_model(model)
+def get_pii_stats(save=False):
+    """
+    Determines the PII stats for all models and fields. Takes existing
+    data from gdpr.yml into account if it exists. If save is True, the
+    data is saved back to gdpr.yml.
 
-    def apply_existing_input_data(self, input_data):
-        for model_label in self.models:
-            for field_label in self.models[model_label]["fields"]:
-                self.models[model_label]["fields"][field_label].update(
-                    get_manual_input_for_field(input_data, model_label, field_label)
-                )
+    :returns Counter: The PII stats, as a Counter with three keys: None, True
+        and False. The None key contains all fields that have not been
+        classified as PII or not. The True key contains all fields that have
+        been classified as PII. The False key contains all fields that have
+        been classified as non-PII.
+    """
+    data = read_data()
+    # migration ignore -> exclude: accept both, save as 'exclude'
+    exclude_list = data.get("exclude", []) + data.get("ignore", [])
```

Comment:
Shouldn't this pick either `exclude` **or** `ignore` (preferring the first) instead of merging both?
```suggestion
    exclude_list = data.get("exclude", data.get("ignore", []))
```